runtime/imports.go

```diff
@@ -199,15 +199,47 @@ func ext_set_storage(context unsafe.Pointer, keyData, keyLen, valueData, valueLe
 
 //export ext_set_child_storage
 func ext_set_child_storage(context unsafe.Pointer, storageKeyData, storageKeyLen, keyData, keyLen, valueData, valueLen int32) {
-	log.Debug("[ext_ed25519_sign] executing...")
-	log.Warn("[ext_ed25519_sign] Not yet implemented.")
+	log.Debug("[ext_set_child_storage] executing...")
```

Comment:
Please use trace for these, we've updated all logs except the messages from `ext_print_...`

Reply:
done